docs/config/README.md

```diff
@@ -650,6 +651,41 @@ By default the module will be shown if any of the following conditions are met:
 format = "via [🔰 $version](bold red) "
 ```
 
+## Deno
+
+The `deno` module shows you your currently installed version of Deno.
+By default the module will be shown if any of the following conditions are met:
+- The current directory contains a `mod.ts`, `mod.js`, `deps.ts` or `deps.js` file
+
+### Options
+
+| Option              | Default                                           | Description                                     |
+| ------------------- | ------------------------------------------------- | ----------------------------------------------- |
+| `format`            | `"via [$symbol($version )]($style)"`              | The format for the module.                      |
+| `symbol`            | `"🦕  "`                                           | A format string representing the symbol of Deno |
```

Comment:
```suggestion
| `symbol`            | `"🦕 "`                                           | A format string representing the symbol of Deno |
```

Reply:
I know @davidkna requested this earlier https://github.com/starship/starship/pull/2565#discussion_r610603565 but all other emoji symbols have a single trailing space (except gcloud and git tag that have two). Plus this got changed but the code stayed the same.